### PR TITLE
Fixed: Navbar Overflow issue on smaller mobile devices.

### LINKED ISSUE #7917: [BUG] Mobile UI alignment or sizing
### Description

When viewing LinkFree via mobile, the UI doesn't entirely take up the entire screen or isn't aligned to the centre.

### Screenshots

Screenshot_20230702_134020_Chrome
Screenshot_20230702_134025_Chrome
Screenshot_20230702_134031_Chrome

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by hi-Kartik2004**
## Fixes Issue
Closes #7917

## Changes proposed
Fixed the mobile navbar overflowing issue under width 300px, causing the Ui to break for smaller mobile devices.

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

### In production
image

image

### After merging this pull request (PR)
image

image

## Note to reviewers
There is a z-index problem in the mobile navbar but there already exists a issue on this. Refer #8507 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8552"></a>

### CODE REVIEW

**Comment by hi-Kartik2004** [comment]
There seems to be merge conflict, could some one confirm if

```bash
   <label htmlFor={name} className={`sr-only ${className}`}>     -> Current changes.
    <label htmlFor={value} className={`sr-only ${className} hidden md:block`}>  -> When I raised this PR
```

The difference is htmlFor = {`name`} or htmlFor = {`value`}, so could you confirm which change should I keep.

-Kartikeya Saini

**Review by yassineatik — COMMENTED** [review]
It looks well for me , maybe you just need to update your fork fork branch to resolve conflicts .

**Comment by hi-Kartik2004** [comment]
I have resolved the merge conflict by keeping both changes.

```bash
     <label htmlFor={name} className={`sr-only ${className}`}>     -> Changes in production.
    <label htmlFor={value} className={`sr-only ${className} hidden md:block`}>  -> When I raised this PR.

    <label htmlFor={name} className={`sr-only ${className} hidden md:block`}>  -> After resolving the conflict.
```

File: components/navbar/Navbar.js [hunk 1]
```diff
@@ -242,7 +242,7 @@ export default function Navbar() {
           </div>
           <div className="pt-4 pb-3 border-t border-primary-medium">
             <div className="flex items-center px-5">
-              <div className="flex items-center md:ml-6">
+              <div className="flex items-center md:ml-6 flex-wrap">
```
Comment (ThomasCode92) [inline components/navbar/Navbar.js:245]: Maybe you can center this? Centered items looks a bit nicer, imo.
image

**Review by ThomasCode92 — APPROVED** [review]

**Comment by azhechkova** [comment]
Looks good to me

**Review by azhechkova — APPROVED** [review]

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**